tests/Converters/JsonConverterShould.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FluentAssertions;
+using Kros.KORM.Converter;
+using System;
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace Kros.KORM.Extensions.Api.UnitTests.Converters
+{
+    public class JsonConverterShould
+    {
+        [Fact]
+        public void ThrowArgumentExceptionWhenEntityTypeIsNotSet()
+        {
+            Action action = () => new JsonConverter(null);
+
+            action.Should().Throw<ArgumentException>();
+        }
+
+        [Fact]
+        public void ConvertJsonToEntity()
+        {
+            var converter = new JsonConverter(typeof(TestClass));
+
+            TestClass expected = GetSampleClass();
+            var actual = converter.Convert(GetSampleJson());
+
+            actual.Should().BeOfType(typeof(TestClass));
+            ((TestClass)actual).BoolProperty.Should().Equals(expected.BoolProperty);
```

Comment:
FluentAssertions can [compare objects](https://fluentassertions.com/objectgraphs/).
```
((TestClass)actual).Should().BeEquivalentTo(expected);
```

Reply:
I did not know that, nice.